### ISSUE TITLE
Please add a GitHub Discussion

### ISSUE THREAD

**Issue by cclauss** [omz/Pythonista-Issues#726]
As discussed at https://forum.omz-software.com/topic/14510
@omz

**Comment by mrlucmorin** [comment]
This is more relevant than ever, as Ole seems to have closed his user forum on omz-software.

@omz, what are your intentions for the future of Pythonista? I, for one, would be willing to pay for continuous upgrades if that's what is causing you to have second thoughts about maintaining further. I'm sure others would follow suit in order to keep the product alive, and see even more features added.

**Comment by ikiziki** [comment]
I'd gladly pay all over again for feature updates. I like the working copy pricing structure requiring a new purchase for major upgrades while keeping smaller ones included for life. 

Please let this app live on. I search almost daily to see if there's any news about an unexpected update again.

**Comment by cclauss** [comment]
iOS is a "fully supported platform" in Python 3.13 which is on its last beta release and should be released in October 2024.
* https://docs.python.org/3.13/whatsnew/3.13.html#support-for-mobile-platforms

This means there should be a lot of new Python-on-iOS innovation in the near future.

**Comment by mrlucmorin** [comment]
@cclauss , I was talking about this on IRC, and apparently the iOS support is not really in terms of being able to develop on iOS. Apparently the same restrictions will exist as far as importing non-Python modules.

Do you have more information on this Python update, but specifically about what it will mean for us who want to develop on an iPad?

Thanks

**Comment by cclauss** [comment]
The leader
of this effort manages https://beeware.org

Given that Pythonista and Pyto.app have demonstrated the potential and power of development on mobile devices, this is an area where we will see continued innovation.

Compiling programs on iOS is more blocked by Apple’s security guidelines which I doubt will change.

**Comment by mrlucmorin** [comment]
Agreed. Apple is not going to allow any competition to XCode. While I do like their products, their policies irk me at the extreme!

As if they're going to lose so many customers. People who already embrace the Mac train will keep doing so, but people like myself who have to use Windows at a professional level will not spend extra thousands just to be allowed to use XCode. No thanks. 

The engineering software that I have to use in my every day work are Windows only. If Apple allowed me to license their OS and run it on VMWare, I'd be all in. But they want to sell the hardware alongside, so no way José.

**Comment by mdxs** [comment]
@cclauss : It appears to me that https://github.com/omz/Pythonista/discussions replaced the old [forum](https://forum.omz-software.com/category/5/pythonista); thus this issue (originally) about "Please add a GitHub Discussion" has been fulfilled and can be closed (and people here with outstanding questions/discussions can move it to the new discussion area).

**Comment by omz** [comment]
@mdxs You're right. Here's a link to the new forum on GitHub: https://github.com/omz/Pythonista/discussions
